### PR TITLE
added doc to envis_trainer.py

### PR DESCRIPTION
**Pull request by bashirk**
## Description
This PR adds inline documentation to the envis_trainer.py file.

## Relevant Issue
Issue #269 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)

### CODE REVIEW

File: fedrec/user_modules/envis_trainer.py [hunk 1]
```diff
@@ -41,6 +89,50 @@ def check_only_one_declaration(instance, _, value):
 
 
 class EnvisTrainer(EnvisBase):
+    """
+    This class is used to train the model. It is responsible for training the
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_trainer.py:93]:
```suggestion
    EnvisTrainer is used to train the model. It is responsible for training the
```
Please don't use `responsible` every time. You can come up with another alternative like `in charge of` and some phrases like that are more technical.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/user_modules/envis_trainer.py [hunk 2]
```diff
@@ -14,6 +14,54 @@
 
 @attr.s
 class TrainConfig:
+    """
+    The TrainConfig class stores the training configuration for the model. It
+    is used to pass the configuration to the trainer. The trainer uses the
+    configuration to train the model. The trainer also uses the configuration
```
Comment (divyanshugit) [inline fedrec/user_modules/envis_trainer.py:20]:
```suggestion
    configuration to training the model. The trainer uses the same configuration
```

**Review by divyanshugit — CHANGES_REQUESTED** [review]

**Comment by divyanshugit** [comment]
LGTM!! @ramesht007

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Pull request merged**